apps/next/src/app/components/nav/nav.html

```diff
@@ -5,12 +5,12 @@
     </a>
     <nav class="next-nav-bar">
       <ul class="next-nav-list">
-        <li class="next-nav-item" *ngFor="let item of (_navData$)?.navItems">
+        <li class="next-nav-item" *ngFor="let item of _navItems">
           <a
             class="next-nav-link"
-            [routerLink]="item.url"
+            [routerLink]="item.toLowerCase()"
```

Comment:
Please prepare the item in the typescript file, logic like toLowerCase should not be handled in the template.
```suggestion
            [routerLink]="item.toLowerCase()"
```